src/tree/document.rs

```diff
@@ -171,26 +213,51 @@ impl Document {
   }
 
   /// Serializes the `Document`
-  pub fn to_string(&self, format: bool) -> String {
+  pub fn to_string(&self) -> String {
```

Comment:
You can entirely remove the `to_string` definition here, it is auto-generated by the `fmt::Display` implementation

Reply:
Done